Assets/Scripts/ResourceLogic.cs

```diff
@@ -0,0 +1,155 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEditor.UIElements;
+using UnityEngine;
+
+public class ResourceLogic : MonoBehaviour
+{
+    //values are out of 100
+    private int paperSupply;
+    private int citizenHappiness;
+    private int money;
+    //value is out of 5 with a HIGHER VALUE = SHORTER GAME!!
+    private int lengthOfGame;
+    private Boolean hasSetLength;
+
+
+    // Start is called before the first frame update
+    void Awake()
+    {
+        paperSupply = 50;
+        citizenHappiness = 50;
+        money = 50;
+        lengthOfGame = 2;
+        hasSetLength = false;
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+
+
+    }
+
+
+
+    //MODIFY VALUES SHOULD BE -3  ->  3 TO MAKE GAME LENGTH APPROPRIATE
+
+    public void modifyPaperSupply(int amount)
+    {
+        if (amount<0 && paperSupply + amount * lengthOfGame >= 0)
```

Comment:
This calculation is done many times in the code below - sometimes it helps teammates (and your future self) follow your logic when you extract this to a variable and name it descriptively (so people aren't concentrating as much on how the calculation is determined vs what it represents).

Reply:
Sorry, I am a little confused what you mean by this. Is there any way you could clarify what I should extract to a variable and how you would recommend I do that most effectively? Thank you!

Reply:
Probably make a function for it I'm guessing?